app/helpers/admin_helper.rb

```diff
@@ -14,4 +14,20 @@ def display_name(group_name)
   def dom_safe_name(group_name)
     group_name.gsub(/\s+|\./, "_").gsub(/\A(\d)/, '_\1')
   end
+
+  def current?(request, group, group_name)
+    deduced_scope(request).to_s == (group[:children][0][:parent] || group_name).to_s
+  end
+
+  def children?(group)
+    group[:children].length == 1
```

Comment:
Just double-checking as I'm not 100% sure what the data structure looks like, but we really want `== 1` and not `>= 1`, right?

Reply:
I think this helper name was a bit misleading here and its usage in `_nested_sidebar.html.erb` was off too.. I changed that logic a little so I think it makes more sense now :). But that's a good catch, thanks!